subt/docker/build.bash

```diff
@@ -36,15 +36,15 @@ fi
 
 user_id=$(id -u)
 image_name=$(basename $1)
-image_plus_tag=$image_name:$(date +%Y_%b_%d_%H%M)
+image_plus_tag=$image_name:$(date +%Y_%m_%d_%H%M)
```

Comment:
well, at this moment this is not ideal change ... i.e. changing docker names in the middle (or rather the end) ... but I see your motivation

Reply:
`%b` is locale specific and while it works this month, it has problems with červen, červenec etc. due to the special characters. I had this change for a long time in some other build scripts so I just brought it over here as well.

Reply:
OK, I see - I would not expect Czech install - OK, keep this change